features/support/hooks.js

```diff
@@ -1,14 +1,15 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { Before, After } from 'cucumber'
```

Comment:
Is this file needed?

Reply:
Yes, and it should setup/teardown the environment. This should be done in another PR though